.gitignore

```diff
@@ -41,3 +42,5 @@ prefabs.json
 /pokeemerald-*.png
 src/data/map_group_count.h
 tools/trainerproc/trainerproc
+.gitignore
+mgba-rom-test
```

Comment:
missing new line but please figure out in general with this is included in the pr in general.  This shouldn't happen.

Reply:
i believe the main reason it's included is that visual studio puts a bunch of stuff in .vs/ and it needs to be excluded before commit. i'm unsure of how to reconcile this, maybe by deleting the folder in between commits?

Reply:
would it be possible to add `vs/` to the project's `.gitignore` file? probably in a separate PR